project2/styles.css

```diff
@@ -0,0 +1,121 @@
+* {
+    box-sizing: border-box;
+    transition: all 1s;
+}
+
+body {
+    background-color: #111111;
+    color: e0e0e0;
+    font-family: Rubik, Sans-serif;
+    font-weight: 400;
+}
+
+.info {
+    background-color: #2b2b2b;
+    color: #e0e0e0;
+    max-width: 500px;
+    max-height: 350px;
+    padding: 15px;
+    position: relative;
+    top: 30px;
+    left: 425px;
+
+}
+
+img {
+    max-width: 500px;
+    position: relative;
+    top: 40px;
+    left: 425px;
+}
+
+h2 {
+    font-size: 30px;
+    font-weight: 450;
```

Comment:
`font-weight` values must be hundred values: 100, 200, 300, 400, etc. So, this could be `400` or `500`, but not `450`.